docs/src/main/paradox/general/configuration-reference.md

```diff
@@ -3,8 +3,7 @@
 Each Pekko module has a `reference.conf` file with the default values.
 
 Make your edits/overrides in your `application.conf`. Don't override default values if
-you are not sure of the implications. [Pekko Config Checker](https://doc.akka.io/docs/akka-enhancements/current/config-checker.html)
-is a useful tool for finding potential configuration issues.
+you are not sure of the implications. Pekko Config Checker is a useful tool for finding potential configuration issues.
```

Comment:
There is no such thing as a `Pekko Config Checker`

Reply:
Understood, let me remove the line.